meta/group-info/th.yaml

```
@@ -0,0 +1,63 @@
+separator: ", "
+alumni:
+  roles:
+    academy: ผู้ค้นพบ osu!academy
+    admin: ผู้บริหารจัดการ
+    chat: ผู้ดูแลแชท
+    dev: ผู้พัฒนา osu!
+    media: ผู้สร้างสื่อ osu!
+    support: ทีมช่วยเหลือ
+    tournaments: ทัวร์นาเมนต์
+gmt:
+  all_mods: ผู้ดูแลทั้งหมด
+  areas:
+    beatmap: ผู้ดูแลบีทแมพ
+    chat: ผู้ดูแลการแชท
+    forum: ผู้ดูแลฟอรั่ม
+    team: ทีมอย่างเป็นทางการ
+    dev: ผู้พัฒนา osu!
+    osu_idol: เจ้าภาพ osu!idol
+    player: ผู้สนับสนุนผู้เล่น
+    support: ผู้สนับสนุนทางเทคนิค
+    tournaments: ทัวร์นาเมนต์
+    wiki: ผู้บริหารวิกิ
```

```suggestion
    wiki: ผู้ควบคุมวิกิ
```
Just a suggestion. The ผู้บริหาร is also ok

---

Updated to 'ผู้ดูแลวิกิ'